waku/waku_archive/retention_policy/builder.nim

```diff
@@ -51,5 +52,38 @@ proc new*(T: type RetentionPolicy,
     let retPolicy: RetentionPolicy = CapacityRetentionPolicy.init(retentionCapacity)
     return ok(some(retPolicy))
 
+  elif policy == "size":
```

Comment:
Unrelated but why is this not a enum instead of a string?

Reply:
overkill for limited parameters?